### ISSUE TITLE
After Archive and install，'This app cannot be installed because its integrity could not be verified'

### ISSUE THREAD

**Issue by wuyubeichen** [onevcat/Kingfisher#1865]
### Check List

Thanks for considering to open an issue. Before you submit your issue, please confirm these boxes are checked.

- [x] I have read the [wiki page](https://github.com/onevcat/Kingfisher/wiki) and [cheat sheet](https://github.com/onevcat/Kingfisher/wiki/Cheat-Sheet), but there is no information I need.
- [x] I have searched in [existing issues](https://github.com/onevcat/Kingfisher/issues?utf8=✓&q=is%3Aissue), but did not find a same one.
- [x] I want to report a problem instead of asking a question. It'd better to use [kingfisher tag in Stack Overflow](http://stackoverflow.com/questions/tagged/kingfisher) to ask a question.

### Issue Description

After Archive and install，I can’t open my App： "This app cannot be installed because its integrity could not be verified".

This problem has only arisen since I used version 7.1.1。And I tried Version 6-Xcode13, still not.

The previous version was no problem.

When open the Kingfisher.framework , executable file is not exists.
![](https://user-images.githubusercontent.com/13478778/140255719-39a90673-7243-487c-8350-496c0da10e39.png)

**Comment by onevcat** [comment]
How did you install Kingfisher? I guess it is not an issue of Kingfisher itself (since we are not providing a prebuilt framework), but a problem with the dependency manager you are using or the build process of your app.

**Comment by DreamcoffeeZS** [comment]
> How did you install Kingfisher? I guess it is not an issue of Kingfisher itself (since we are not providing a prebuilt framework), but a problem with the dependency manager you are using or the build process of your app.

Thank you very much for your reply!

I installed Kingfisher by CocoaPods.

In order to find the problem，after archive，I find the .ipa file -> displayed the package contents -> Frameworks -> Kingfisher.framework.

Then， I saw the screenshot， I think this is unusual.

Finally I have to gived up  this method，and  solved the problem by using "Swift Package Manager”.

But，I don't know exactly what caused it.
